centreon/packaging/debian/centreon-poller-centreon-engine.postinst

```diff
@@ -2,6 +2,13 @@
 
 if [ "$1" = "configure" ]; then
 
+    chmod 0774 /var/log/centreon
+    chmod 0775 /var/run/centreon
```

Comment:
Not a 750 on /var/run/centreon, like on centreon/packaging/centreon.spectemplate +436 ?

Reply:
750 is in the `%files` section, and overrided to `0775` in the `%install` section